src/components/application_manager/src/commands/mobile/subscribe_button_request.cc

```diff
@@ -58,7 +58,7 @@ void SubscribeButtonRequest::Run() {
 
   const mobile_apis::ButtonName::eType btn_id =
```

Comment:
@wolfylambova22 , we will have a meeting in the near future about coding style. Now we name local const without `k`